libraries/wallet/wallet.cpp

```diff
@@ -205,9 +205,13 @@ namespace golos { namespace wallet {
 
             public:
                 wallet_api& self;
-                wallet_api_impl( wallet_api& s, const wallet_data& initial_data, const golos::protocol::chain_id_type& _steem_chain_id, fc::api< remote_node_api > rapi )
-                        : self( s ),
-                          _remote_api( rapi )
+                wallet_api_impl( wallet_api& s, const wallet_data& initial_data, const golos::protocol::chain_id_type& _steem_chain_id, fc::api_connection& con ):
+                    self( s ),
+                    _remote_database_api( con.get_remote_api< golos::wallet::remote_database_api >( 0, "database_api" ) ),
```

Comment:
Might be a fine idea to store names of plugins "database_api" ... somwhere and not hardcode every time.